scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts

```diff
@@ -229,6 +231,14 @@ export function ContentProfileSchemaEditor(content, metadata, vocabularies) {
                  * @param {String} id the key of the field to toggle.
                  */
                 scope.toggle = (schema, order, position) => {
+                    let newOrder = order;
+
+                    if (scope.model.editor[schema.key].enabled) {
+                        notify.error(
+                            gettext(`This item is already added in ${scope.model.editor[schema.key].section} fields.`),
```

Comment:
please use the . following format for gettext:
>gettext('This item is already added in {{section}} fields.', {section: scope.model.editor[schema.key].section});

Reply:
@ioanpocol  fixed that.